frontend/src/app.tsx

```diff
@@ -31,9 +32,13 @@ const App: React.FC = () => (
       component={Users}
       permissions={[PermissionName.CREATE_USER, PermissionName.EDIT_USER]}
     />
-    <AuthorizedRoute exact path={AppRoute.MAP} component={Map}
+    <AuthorizedRoute
+      exact
+      path={AppRoute.MAP}
+      component={Map}
       permissions={[PermissionName.MAP_MANIPULATION]}
     />
+    <Route path={AppRoute.EDIT_PERMISSIONS} component={Permissions} />
```

Comment:
```suggestion
    < AuthorizedRoute path={AppRoute.EDIT_PERMISSIONS} component={Permissions} />
```
Don't forget to add permission for this route